setup.py

```diff
@@ -95,7 +95,7 @@ def build_py_modules(basedir, excludes=[]):
     license="BSD",
     url="https://github.com/gitpython-developers/GitPython",
     packages=find_packages(exclude=("test.*")),
-    package_data={'git': ['**/*.pyi', 'py.typed']},
+    # package_data={'git': ['**/*.pyi', 'py.typed']},
```

Comment:
Is this something that can be removed?

Reply:
As far as I can tell, we should use package_data OR use_package_data.
Use_package_data=True looks at MANIFEST.INI for files and ignores files listed in package_data.
(like here: https://github.com/encode/httpx/issues/193#issue-477224461)

So as we already have use_package_data=True, i've moved the py.typed entry to MANIFEST.INI.
I'll try a fresh install from github once merged and see if it then picks up the py.typed.

Reply:
Ok, I see that this is to be removed once it's clear that the MANIFEST.INI  will work as expected also in conjunction with 'py.typed'.